GoMain/bin/res/sample-json-device.yaml

```diff
@@ -0,0 +1,61 @@
+name: "7e841a12-a083-48fe-8040-83815e0b7410"
```

Comment:
@suresh-lc Same opinion as `GoMain/bin/res/configuration.toml` and `GoMain/bin/res/docker/configuration.toml`.